specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/securityContacts.json

```diff
@@ -0,0 +1,366 @@
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/securityContacts": {
+      "get": {
+        "x-ms-examples": {
+          "List security contact data": {
+            "$ref": "./examples/SecurityContacts/GetSecurityContactsSubscription_example.json"
+          }
+        },
+        "tags": [
+          "Security Contacts"
+        ],
+        "description": "List all security contact configurations for the subscription",
+        "operationId": "SecurityContacts_List",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecurityContactList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/securityContacts/default": {
+      "get": {
+        "x-ms-examples": {
+          "Get default security contact": {
+            "$ref": "./examples/SecurityContacts/GetSecurityContact_example.json"
+          }
+        },
+        "tags": [
+          "Security Contacts"
+        ],
+        "description": "Get Default Security contact configurations for the subscription",
+        "operationId": "SecurityContacts_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecurityContact"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Create security contact data minimal": {
```

Comment:
minor: I'd suggest to remove the "minimal", unless there is another put request documented, for full model